crates/iota/src/name_commands.rs

```diff
@@ -1888,7 +1882,10 @@ async fn get_proxy_nft_by_name(
     })
 }
 
-async fn get_registry_entry(domain: &Domain, client: &IotaClient) -> anyhow::Result<RegistryEntry> {
+async fn get_registry_entry(
+    domain: &Domain,
+    client: &IotaClient,
+) -> Result<RegistryEntry, RpcError> {
```

Comment:
Why change the result?

Reply:
Because the return of `get_object_from_bcs` changed but I guess I could map this one

Reply:
why did they change in the first place?

Reply:
Because there is two layers of error coming from the API, an actual error variant and on success case an object that can contain an error. We can't properly match that wrapped error if we keep a generic anyhow error

Reply:
That's kind of what I'm doing? There are 2 sources or error so I'm combining them into one so the caller can properly handle it